ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriber.java

```diff
@@ -75,12 +75,13 @@ public void onError(final Throwable e) {
     final String msg = e.getMessage() == null || e.getMessage().trim().isEmpty()
         ? "KSQL exception: " + e.getClass().getSimpleName()
         : e.getMessage();
-
+    close();
```

Comment:
Since `SessionUtil.closeSilently()` invokes `close()`, this seems unnecessary. It also would send the client the default close frame code, rather than the specific code currently passed to `closeSilently`. Same below in `onComplete`.

Reply:
Done!